src/tests/system/tests/test_offline.py

```diff
@@ -0,0 +1,91 @@
+"""
+Automation of offline tests
+
+:requirement: offline
+"""
+
+from __future__ import annotations
+
+import time
+
+import pytest
+from sssd_test_framework.roles.client import Client
+from sssd_test_framework.roles.ldap import LDAP
+from sssd_test_framework.topology import KnownTopology
+
+
+@pytest.mark.ticket(bz=1416150)
+@pytest.mark.topology(KnownTopology.LDAP)
+def test_offline__log_to_syslog(client: Client):
+    """
+    :title: Log to syslog when sssd cannot contact servers goes offline
+    :setup:
+        1. Set an invalid hostname uri and disable the offset to refresh sudo rules
+        2. Start SSSD
+    :steps:
+        1. Check domain status for default domain
+        2. Clear journal and restart SSSD
+        3. Check journalctl
+    :expectedresults:
+        1. Domain is offline
+        2. Succeed
+        3. "Backend is offline" found
+    :customerscenario: True
+    """
+    client.sssd.domain["ldap_uri"] = "ldaps://typo.invalid"
+    client.sssd.domain["ldap_sudo_random_offset"] = "0"
+    client.sssd.start()
+    assert client.sssd.default_domain is not None, "Failed to load default domain"
+    status = client.sssctl.domain_status(client.sssd.default_domain)
+    assert "Offline" in status.stdout or "Unable to get online status" in status.stderr, "Domain is not offline"
+
+    client.journald.clear()
+    client.sssd.restart()
+    time.sleep(1)
+
+    log = client.journald.journalctl(grep="Backend is offline", unit="sssd")
+    assert log.rc == 0, "'Backend is offline' is not logged"
+
+
+@pytest.mark.ticket(bz=1928648)
+@pytest.mark.topology(KnownTopology.LDAP)
+def test_offline__timeout_setting_in_logs(client: Client, ldap: LDAP):
```

Comment:
I'd like to see ldap in the test name, since this only applies to LDAP. 

test_offline__ldap_network_timeout_parameters_shown_in_logs() ?

Reply:
Fixed